fedora/jellyfin.spec

```diff
@@ -131,20 +106,20 @@ if [ $1 -gt 1 ] ; then
     if [ "${service_state}" = "active" ]; then
         systemctl stop jellyfin.service
     fi
-    if [ ! -L %{_sharedstatedir}/%{name}/config ]; then
-        mv %{_sharedstatedir}/%{name}/config/* %{_sysconfdir}/%{name}/
-        rmdir %{_sharedstatedir}/%{name}/config
-        ln -sf %{_sysconfdir}/%{name}  %{_sharedstatedir}/%{name}/config
+    if [ ! -L %{_sharedstatedir}/jellyfin/config ]; then
+        mv %{_sharedstatedir}/jellyfin/config/* %{_sysconfdir}/jellyfin/
+        rmdir %{_sharedstatedir}/jellyfin/config
+        ln -sf %{_sysconfdir}/jellyfin  %{_sharedstatedir}/jellyfin/config
     fi
-    if [ ! -L %{_sharedstatedir}/%{name}/logs ]; then
-        mv %{_sharedstatedir}/%{name}/logs/* %{_var}/log/jellyfin
-        rmdir %{_sharedstatedir}/%{name}/logs
-        ln -sf %{_var}/log/jellyfin  %{_sharedstatedir}/%{name}/logs
+    if [ ! -L %{_sharedstatedir}/jellyfin/logs ]; then
+        mv %{_sharedstatedir}/jellyfin/logs/* %{_var}/log/jellyfin
+        rmdir %{_sharedstatedir}/jellyfin/logs
+        ln -sf %{_var}/log/jellyfin  %{_sharedstatedir}/jellyfin/logs
     fi
-    if [ ! -L %{_sharedstatedir}/%{name}/cache ]; then
-        mv %{_sharedstatedir}/%{name}/cache/* %{_var}/cache/jellyfin
-        rmdir %{_sharedstatedir}/%{name}/cache
-        ln -sf %{_var}/cache/jellyfin  %{_sharedstatedir}/%{name}/cache
+    if [ ! -L %{_sharedstatedir}/jellyfin/cache ]; then
+        mv %{_sharedstatedir}/jellyfin/cache/* %{_var}/cache/jellyfin
+        rmdir %{_sharedstatedir}/jellyfin/cache
+        ln -sf %{_var}/cache/jellyfin  %{_sharedstatedir}/jellyfin/cache
```

Comment:
Ditto.

Reply:
ibid. - Files will still be "jellyfin" rather than "jellyfin-server", just the package name should be altered.

Reply:
Fair enough.